client/jetbrains/src/main/kotlin/com/sourcegraph/cody/autocomplete/render/CodyAutocompleteSingleLineRenderer.kt

```diff
@@ -0,0 +1,28 @@
+package com.sourcegraph.cody.autocomplete.render
+
+import com.intellij.openapi.editor.Editor
+import com.intellij.openapi.editor.Inlay
+import com.intellij.openapi.editor.markup.TextAttributes
+import com.sourcegraph.cody.vscode.InlineAutocompleteItem
+import java.awt.Graphics
+import java.awt.Rectangle
+
+class CodyAutocompleteSingleLineRenderer(
+    text: String?,
+    completionItem: InlineAutocompleteItem?,
+    editor: Editor?,
+    type: AutocompleteRendererType?
+) : CodyAutocompleteElementRenderer(text!!, completionItem, editor!!, type) {
```

Comment:
Had a quick look at instantiation sites, and we can remove the nullability on `text` and `editor` here
```suggestion
class CodyAutocompleteSingleLineRenderer(
    text: String,
    completionItem: InlineAutocompleteItem?,
    editor: Editor,
    type: AutocompleteRendererType?
) : CodyAutocompleteElementRenderer(text, completionItem, editor, type) {
```

Reply:
I _think_ we can make `type` non-nullable too, have edited the suggestion.

Reply:
indeed we can.
it's so nice to let the Kotlin compiler figure nullability for us.
fixed